Source/WebCore/svg/SVGLengthContext.cpp

```diff
@@ -233,6 +237,19 @@ static inline const RenderStyle* renderStyleForLengthResolving(const SVGElement*
     return nullptr;
 }
 
+static inline const RenderStyle* getRootRenderStyleForLength(const SVGElement* svgElement)
```

Comment:
```suggestion
static inline const RenderStyle* rootRenderStyleForLength(const SVGElement* svgElement)
```

nit: we don't use the `get` prefix in the webkit codebase